changelog.d/2037.change.rst

```diff
@@ -0,0 +1,4 @@
+A bit "cosmetic" changes though needed for implementing more advanced stuff.
+* Renamed args names params for some static methods and marked them with ''@staticmethod''.
+* Moved ''0'' into ''default_order_value" variable.
```

Comment:
I'd agree with this change to provide more context. The problem is that the purpose of the default value is already documented in the docstring, so with this change, it's now documented redundantly, opening up the opportunity for the two definitions to diverge. Especially if the code for the function remains small (< 10 lines), I find it unnecessary to make the code self-documenting here.

Reply:
It was done to just make [the diff](https://github.com/pypa/setuptools/pull/2038/files#diff-0f33d7d69d9f143736a252a75373261bR791) of  #2038  a bit cleaner.